scripts/train_lookup_lemmatizer.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+
+import typer
+
+from tqdm import tqdm
+from conllu import parse_incr
+
+from spacy.lookups import Lookups, Table
+
+app = typer.Typer()
+
+
+@app.command()
+def main(input_file: str, output_path: Path, min_occurrences: int = 1):
+    token_lemma_occurrences = dict()
+
+    with open(input_file) as inp:
+        for sentence in tqdm(list(parse_incr(inp))):
+            for token in sentence:
+                if token["upos"] not in token_lemma_occurrences.keys():
```

Comment:
This logic can be simplified with `collections.Defaultdict` or `collections.Counter`. I'd suggest to use the latter one.

Reply:
Replaced!